dragn/tests/test_generated_dice.py

```diff
@@ -0,0 +1,21 @@
+import pytest
+
+
+from dragn import dice
+
+
+class TestModuleGetattr:
```

Comment:
We don't need this extra file for tests, if you remove the fixed builders on `dice/__init__.py`, the `test_dice.py` will be testing the `__getattr__` automatically.

So, could you just integrate the invalid cases into the same file too?

Reply:
That's a good idea, I'll make the changes.